django/contrib/admindocs/templates/admin_doc/template_detail.html

```diff
@@ -5,8 +5,8 @@
 <div class="breadcrumbs">
 <a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
 &rsaquo; <a href="{% url 'django-admindocs-docroot' %}">{% translate 'Documentation' %}</a>
-&rsaquo; {% translate 'Templates' %}
-&rsaquo; {{ name }}
+&rsaquo; <a href="#">{% translate 'Templates' %}</a>
+&rsaquo; <a href="#" aria-current="page">{{ name }}</a>
```

Comment:
Maybe something like the following would make more sense if there is no "Templates" page that can be navigated to?

```suggestion
&rsaquo; <a href="#" aria-current="page">{% blocktranslate with name=name %}Template: {{ name }}{% endblocktranslate %}</a>
```